schemas/iio/adc.yaml

```diff
@@ -0,0 +1,40 @@
+# SPDX-License-Identifier: GPL-2.0
```

Comment:
Dual license. I want everything in dtschema BSD at least. To the point that I've added some schemas without their descriptions because I haven't gone thru the relicensing exercise.

Reply:
License thing was accidental.  Guess I cut and paste from an older file in the kernel tree.